requirements/lite/base.txt

```diff
@@ -7,3 +7,4 @@ fsspec[http]>2021.06.0, <2022.6.0
 packaging>=17.0, <=21.3
 typing-extensions>=4.0.0, <=4.4.0
 lightning-utilities==0.3.*
+click>=8.0.0, <=8.1.3
```

Comment:
I don't think this should be added here. Lite should still work without click installed. The `CLI` interface is not a requirement for it's use.

Reply:
If click is not a requirement, what should I do? Ask user to install it if they use the CLI? At some point, we will have docs for Lite (and the standalone package) and we will show the command line as the primary way of interaction.